pubsub/hazelcast/hazelcast.go

```diff
@@ -33,6 +44,14 @@ func parseHazelcastMetadata(meta pubsub.Metadata) (metadata, error) {
 		return m, errors.New("hazelcast error: missing hazelcast servers")
 	}
 
+	if val, ok := meta.Properties[hazelcastBackOffMaxRetries]; ok && val != "" {
```

Comment:
Create a doc issue to update this metadata in dapr/docs.

Reply:
Already did dapr/docs/issues/1270